tests/common/macro_support.hpp

```diff
@@ -146,8 +146,8 @@ inline std::vector<Name> sort_names( std::vector<Name>&& names ) {
 #define STAKE4(chain, sender, recipient, amount) \
    { \
       eos::chain::SignedTransaction trx; \
-      trx.scope = sort_names( { #sender, #recipient } ); \
-      trx.emplaceMessage(config::EosContractName, vector<types::AccountPermission>{{#sender, "active"}}, \
+      trx.scope = sort_names( { #sender, #recipient, "eos" } ); \
```

Comment:
It's better to use `config::EosContractName` rather than the literal "eos". I'd like to minimize the number of places that name the contract directly.